makefile

```diff
@@ -173,11 +173,11 @@ dist:$(buildDir)/dist.tar.gz
 dist-test:$(buildDir)/dist-test.tar.gz
 dist-source:$(buildDir)/dist-source.tar.gz
 $(buildDir)/dist.tar.gz:$(buildDir)/make-tarball $(clientBinaries)
-	./$< --name $@ --prefix $(name) $(foreach item,$(distContents),--item $(item))
+	./$< --name $@ --prefix $(name) $(foreach item,$(distContents),--item $(item)) --exclude "public/node_modules"
```

Comment:
In case anyone is wondering, if node_modules have been generated the make-tarball command fails to some header errors, so this is necessary.